src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -263,4 +266,13 @@ private void validateFilingDate(LocalDate filingDate, Errors errors, String logg
             ApiLogger.infoContext(loggingContext, errorMessage);
         }
     }
+
+    private void validateHasSecureRegister(Boolean hasSecureRegister, Errors errors, String loggingContext) {
+        if (Boolean.TRUE.equals(hasSecureRegister) || hasSecureRegister == null) {
```

Comment:
In theory, during partial validation, the field may not be supplied, so this shouldn't be an error (I don't think).

Reply:
Yes, you are right it is registration

Reply:
Removed validation form partial registration

Reply:
So we still need to do some validation if it is partial validation - if 'true' is supplied then this is an error. It's just the 'null' check we don't need to do.